docs/quick-start-guide.md

```diff
@@ -17,7 +17,7 @@ To get started, you’ll need:
 Run the following command to start the dev container and launch a terminal session within it:
 
 ```shell
-docker run --rm -it --name choreo-quick-idp \
+docker run --rm -it --name choreo-quick-idp --pull always \
```

Comment:
Do we need the pull always? I think this is more likely due to how we push the image with retagging, once we have a proper pipeline, this will work fine.

Reply:
Isn't it good to have `--pull always` at the moment?? since we only have a single tag??

Reply:
Removed this, since we will be releasing tags, this won't be an issue..